### PR TITLE
fix json + removal of eval users

### PR DESCRIPTION
**Pull request by odra**
## Additional Information

Removes unused evals users and fix installation when specifying 0 seed users.

## Verification Steps

* Run an usual installation and test the webapp login
* Run an installation with "0" seed users: `ansible-playbook -i inventories/hosts playbooks/install.yml --e 'eval_seed_users_count=0'` - only the admin user (`admin@example.com`) should be created.

### CODE REVIEW

**Comment by JameelB** [comment]
:eyes:

**Comment by maleck13** [comment]
The users should be removed from 3scale also

File: evals/roles/3scale/tasks/users.yml [hunk 1]
```diff
@@ -29,12 +29,6 @@
     status_code: [200]
   when: default_admin_user_xml is defined and default_admin_user_xml.matches is defined
 
-- name: Create evaluation user
```
Comment (odra) [inline evals/roles/3scale/tasks/users.yml]: @maleck13 I assumed none of those eval users should be created, does it make sense or we need some kind of "eval user creation" flag?

Comment (maleck13) [reply evals/roles/3scale/tasks/users.yml]: admin@example.com is ok. But everything else should be decided by the flag that indicates the number of evals users to create ```eval_seed_users_count```

**Pull request merged**